src/bacnet/datalink/dlenv.c

```diff
@@ -643,7 +643,7 @@ void dlenv_init(void)
     if (pEnv) {
         dlmstp_set_max_master(strtol(pEnv, NULL, 0));
     } else {
-        dlmstp_set_max_master(127);
+        dlmstp_set_max_master(7);
```

Comment:
The changes to DLENV appear to be for testing. Please use OS environment variables for changes to the values in dlenv.c module, or some other method (not hard coding).  The values in the 'else' are used when a value has not been specified.